lib/main.dart

```diff
@@ -18,6 +18,7 @@ void _setTransparentStatusBar() {
   SystemChrome.setSystemUIOverlayStyle(const SystemUiOverlayStyle(
     statusBarColor: Colors.transparent,
   ));
+  SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle.light);
```

Comment:
@Shayokh144 Hmmm, there's a bug here, because now the status bar is not transparent anymore

We must refactor it like this:
```suggestion
  SystemChrome.setSystemUIOverlayStyle(SystemUiOverlayStyle.light.copyWith(
    statusBarColor: Colors.transparent,
  ));
```

Reply:
@Tuubz thanks for mentioning